qiskit/circuit/measure.py

```diff
@@ -36,3 +39,62 @@ def broadcast_arguments(self, qargs, cargs):
                 yield qarg, [each_carg]
         else:
             raise CircuitError("register size error")
+
+
+class MeasurePauli(Measure):
```

Comment:
See https://github.com/Qiskit/qiskit-terra/pull/5311/files#r810538319
```suggestion
class MeasurePauli(Instruction):
```

Reply:
Strong +1. MeasurePauli should not be a subclass of Measure.